editors/code/src/commands/inlay_hints.ts

```diff
@@ -29,7 +25,7 @@ export class HintsUpdater {
     public async loadHints(
         editor: vscode.TextEditor | undefined
     ): Promise<void> {
-        if (this.displayHints && editor !== undefined) {
+        if (this.displayHints && editor !== undefined && this.isRustDocument(editor.document)) {
```

Comment:
```rust
-        if (this.displayHints && editor !== undefined && this.isRustDocument(editor.document)) {
+        if (
+            this.displayHints &&
+            editor !== undefined &&
+            this.isRustDocument(editor.document)
+        ) {
```

Reply:
Hm, this one was not updated by the `npm run fix`, how did you get that?

Reply:
I ran `prettier --write **/*.{json,ts}`. I noticed that `package-lock.json` has `prettier 1.17`, while I'm running 1.18.2 (I use Arch btw), but so my version seemed to be a better match for what happens on CI. Feel free to ignore my suggested change if it's not actually needed.

Reply:
Thanks, this suggestion helped.
Looks like the outdated lockfile indeed, I wonder how does the CI ignores this file.